mobile/envoy_build_config/test_extensions.cc

```diff
@@ -12,4 +16,8 @@ void register_test_extensions() {
   Envoy::Extensions::HttpFilters::TestKeyValueStore::forceRegisterTestKeyValueStoreFilterFactory();
   Envoy::Extensions::HttpFilters::TestLogger::forceRegisterFactory();
   Envoy::HttpFilters::TestRead::forceRegisterTestReadFilterFactory();
+  Envoy::Extensions::HttpFilters::Assertion::forceRegisterAssertionFilterFactory();
```

Comment:
Can we preserve alphabetical order in this file? I think that this was the original intention